Data/HashMap/Strict/Base.hs

```diff
@@ -39,6 +39,7 @@ module Data.HashMap.Strict.Base
     , size
     , HM.member
     , HM.lookup
+    , (HM.!?)
```

Comment:
Minor optional nitpick: this is a bit of an odd prefix to stare at in source. Can we just export `(!?)`?

Reply:
So the operator `!?` is just being re-exported from the `Data.HashMap.Base` module which is imported qualified as `HM` (see line R105 below). Following convention here, all re-exported symbols without modification/strict-specific implementations for the `Strict` version are explicitly qualified with `HM.`. We _could_ `import Data.HashMap.Base ((!?))` which would let us just export `(!?)` in the list here, but then that could be misinterpreted as having a custom `!?` operator for the strict module.

Reply:
Makes sense, thanks.